### DISCUSSION TITLE
Finding issues to contribute to

### DISCUSSION THREAD

**Discussion by ArtemkaKun** [vlang/v#19440] [General]
Hi @vlang/v-devs 
I want to inform you that a new section ["Finding issues to contribute to"](https://github.com/vlang/v/blob/master/CONTRIBUTING.md#finding-issues-to-contribute-to) was added to our [CONTRIBUTING.md](https://github.com/vlang/v/blob/master/CONTRIBUTING.md) file.

This section will show you how you can filter issues by likes and tags. 

The notification about voting for an issue with likes will be added to every new issue, so users wouldn't forget about likes, and developers will find what users want more easily.

The likes thingy exists for 2 weeks now and seems it's helpful. Let me know what do you think about it.

Have a nice day ☕

**Comment by ylluminate** [comment]
That is a really cool observation and use case. Thanks for bringing that up!!! I think you actually could just make it link to the actual ordered list by making the link: `https://github.com/vlang/v/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc`

**Reply by ArtemkaKun** [reply]
Thats a great idea, will do that improvement now
